Sources/Common/Core/DataArray/index.js

```diff
@@ -262,9 +262,14 @@ function vtkDataArray(publicAPI, model) {
   };
 
   publicAPI.setData = (typedArray, numberOfComponents) => {
-    model.values = typedArray;
-    model.size = typedArray.length;
-    model.dataType = getDataType(typedArray);
+    if (typedArray) {
```

Comment:
if typedArray is null, you should empty model.values, no ?

Reply:
Added an error log and return so that `values` is not updated to null.

Reply:
Current implementation: null can be passed so values is set to null, size to 0, numberOfComponents to 1.